The patch is to avoid Thunderbird crash in NVidia libGL.

--- thunderbird-115.2.0.orig/comm/mail/installer/package-manifest.in	2023-08-28 19:58:49.000000000 +0200
+++ thunderbird-115.2.0/comm/mail/installer/package-manifest.in	2023-09-15 10:08:04.622815535 +0200
@@ -122,7 +122,6 @@
 #endif
 #endif
 #ifdef MOZ_GTK
-@BINPATH@/glxtest
 @BINPATH@/@DLL_PREFIX@mozgtk@DLL_SUFFIX@
 #ifdef MOZ_WAYLAND
 @BINPATH@/@DLL_PREFIX@mozwayland@DLL_SUFFIX@
